UI/MVUX/src/MVUX/Presentation/FeedViewCommandSample/FeedViewCommandModel.cs

```diff
@@ -0,0 +1,30 @@
+namespace MVUX.Presentation.FeedViewCommandSample;
+
+public partial class FeedViewCommandModel
+{
+	public IListState<Item> Items => ListState.Value(this, () =>
+	{
+		return ImmutableList.Create(
+			new Item { Text = "Item 1" },
+			new Item { Text = "Item 2" },
+			new Item { Text = "Item 3" }
+			);
+	});
+
+	public async ValueTask RemoveItem(Item item)
+	{
+		if (item != null)
+		{
+			await Items.UpdateAsync(currentItems =>
+			{
+				var updatedItems = currentItems.Remove(item); 
+				return updatedItems;
+			});
```

Comment:
I think that we should prefer lighter syntax when possible ... but I can agree that for learning purposes it makes sense to have longer syntax.
```suggestion
			await Items.UpdateAsync(currentItems => currentItems.Remove(item));
```